### DISCUSSION TITLE
Capture from screenshot

### DISCUSSION THREAD

**Discussion by aasimenator** [hyperion-project/hyperion.ng#1212] [Ideas]
Hi,
I have been looking into implementing Hyerprion-NG for my living room setup, I really liked the idea and the visualization that these reactive LED's would bring into the media consumption experience. 

So far what I've seen is that apart from the very obvious individually addressable RGB LED's you need a bunch of other things stuck to the back of your TV, this is something I would like to avoid and I am certain that I am not the only one who would want that clean setup. 
So this is what I thought, I'd be using the RGB LED's along with a D1 Mini with WLED flashed to it so it's accessible over WiFi, so the back of my TV would be very clean and simple. You would not need a HDMI Splitter / converter / capture and all that nonsense, because modern TV's are smart, android based. you can watch content directly on it via NETFLIX or Amazon Prime etc, and the need to connect something via HDMI is minimum. If I am paying a lot of money for a smart tv then I would be using its functions at least most of the time right? e.g. as mentioned above Netflix, Amazon Prime, Youtube, HBO, Disney all that is now available on the TV itself, for all those apps built-in Hyperion-ng will not work. and to me, that is a drawback of the system.

So I need to know if it's possible Hyperion-ng can be configured to be able to read from a file (png. jpg. etc.) on a shared or local drive and do what it does best and change the led colors based on those images. those images that are there will be placed from an application that you install on the Andriod TV that captures anything that's on-screen and saves it to a location now the fps for these images would also be based on the pooling interval that hyperion-ng uses in its video capture from hdmi. 

What do you think?

**Comment by Paulchen-Panther** [comment]
https://github.com/abrenoch/hyperion-android-grabber

**Reply by mattman73** [reply]
From what you are saying Hyerpion is not for you and you would be better off purchasing a TV like this one [https://www.philips.co.uk/c-p/55PUS6814_12/6800-series-4k-uhd-led-smart-tv-with-ambilight-3-sided](url) 
Matt

**Reply by aasimenator** [reply]
> https://github.com/abrenoch/hyperion-android-grabber
Thank you this is what I was looking for 
@mattman73  I think you are crazy to even suggest that, Hyperion can do everything those Ambilight TVs can but better. I was confused because 99% of the videos/projects I saw on youtube were with capture devices and not the android grabber app. I would prefer the grabber app and I thought it would a good feature if it didn't exist, seems like it does, so awesome!
